soc/xtensa/esp32/Kconfig.defconfig

```diff
@@ -54,4 +57,19 @@ config GDBSTUB_BUF_SZ
 
 endif
 
+# Indicates if a console UART is used
+config ESP_CONSOLE_UART
+	bool
+	default $(dt_chosen_enabled,$(DT_CHOSEN_Z_CONSOLE)) && \
+		$(dt_chosen_has_compat,$(DT_CHOSEN_Z_CONSOLE),$(DT_COMPAT_ESPRESSIF_ESP32_UART))
+
+# During system clock initialization this config is used to avoid data lost
+# during APB frequency switching. Due to as ESP32 ROM bug, UART2 is not
+# supported for console output, hence, this case is not handled here.
+config ESP_CONSOLE_UART_NUM
```

Comment:
why not use `help` field here?

Reply:
This config supports HAL Espressif's power management logic to flush the console's FIFO queue if needed. Once it is not meant to be user-tweaked, I am not sure if `help` is really needed.

Reply:
I think `help` is just the natural place where to explain what a Kconfig symbol does.

Reply:
@gmarull, sure, I'll update this.